src/bcrhp/bcrhp/pkg/map_layers/mapbox_spec_json/overlays/nts250k/nts250k.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
This layer is to be removed so we probably don't need to copy it over.

Reply:
The reason why this is copied over is for the backward migration. Same with the other layers that will be removed